fastlane_core/lib/fastlane_core/command_executor.rb

```diff
@@ -43,7 +43,7 @@ def execute(command: nil, print_all: false, print_command: true, error: nil, pre
         end
 
         begin
-          PTY.spawn(command) do |stdin, stdout, pid|
+          system(ENV, command) do |stdin, stdout, pid|
```

Comment:
note that the stdin, stdout, pid block is not used here.... perhaps a util like popen3 would be better